tests/bootstrap.js

```diff
@@ -1,11 +1,6 @@
-// Common test utilities.
-global.assert = require('chai').assert;
```

Comment:
@jodiedoubleday we've always used `assert` from `chai`. All I did was nuke it from `window`.